mkdocs.yml

```diff
@@ -11,6 +11,17 @@ nav:
     - 'Style Guidelines':
         - 'Go': 'style/go.md'
         - 'Markdown': 'style/markdown.md'
+    - 'Knowledge Base':
+        - 'Glossary': 'knowledge_base/glossary.md'
+        - 'Consensus':
+            - 'Avalanche':
+                - 'Concepts': 'knowledge_base/consensus/avalanche/concepts.md'
+                - 'Specification': 'knowledge_base/consensus/avalanche/specification.md'
+            - 'Stellar':
+                - 'Concepts': 'knowledge_base/consensus/stellar/concepts.md'
+                - 'Specification': 'knowledge_base/consensus/stellar/specification.md'
+        - 'Flow': 'knowledge_base/flow/specification.md'
+        - 'Flare' 'knowledge_base/flare/specification.md'
```

Comment:
```suggestion
        - 'Flare': 'knowledge_base/flare/specification.md'
```

Reply:
Missed this, thanks!

Reply:
Fixed https://github.com/optakt/optakt.github.io/pull/13/commits/f9b6f65027bb3d9a542f9b33ae71cdbc7bf2ff8c#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R23-R24